openadapt/db/crud.py

```diff
@@ -479,36 +535,167 @@ def get_new_session() -> sa.orm.Session:
     return Session()
 
 
-def update_video_start_time(
-    recording_timestamp: float, video_start_time: float
-) -> None:
+def update_video_start_time(recording: Recording, video_start_time: float) -> None:
     """Update the video start time of a specific recording.
 
     Args:
-        recording_timestamp (float): The timestamp of the recording to update.
+        recording (Recording): The recording object to update.
         video_start_time (float): The new video start time to set.
     """
-    # Find the recording by its timestamp
-    recording = (
-        db.query(Recording).filter(Recording.timestamp == recording_timestamp).first()
-    )
-
-    if not recording:
-        logger.error(f"No recording found with timestamp {recording_timestamp}.")
-        return
-
     # Update the video start time
     recording.video_start_time = video_start_time
 
+    db.add(recording)
```

Comment:
Is this necessary?

Reply:
Thank you for clarifying. Why does the recording object belong to another session? Can you please document this?

Reply:
wonderful!